data-reconciliation-app/test/unit-test/repositories/kv-repository.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as polyfill from "@microsoft/ccf-app/polyfill.js";
+
+import * as jscrypto from "crypto";
+import {
+  IKeyValueRepository,
+  KeyValueRepository,
+} from "../../../src/repositories/kv-repository";
+import { DataRecord } from "../../../src/models/data-record";
+import { describe, expect, test, beforeEach, afterEach } from "@jest/globals";
+
+describe("Key value pair Repository", () => {
+  let keyValueRepo: IKeyValueRepository<DataRecord>;
+  const userId = jscrypto.randomUUID();
+  const testKey = jscrypto.randomUUID();
+  const testDataRecord: DataRecord = DataRecord.create(testKey, "test");
+
+  beforeEach(() => {
+    keyValueRepo = new KeyValueRepository<DataRecord>();
+  });
+
+  afterEach(() => {});
+
+  test("Should add a new key-value pair", () => {
+    // Act
+
+    // Assert
+    const result = keyValueRepo.set(testKey, testDataRecord);
```

Comment:
I didn't see any business logic at `Repository` which just uses the `kvStore` as is. Do we really need to test it?

Reply:
It is testing the integration with ccf key-value map